.github/workflows/go-tests-other-os.yml

```diff
@@ -12,10 +12,10 @@ jobs:
     name: Test MacOS
     runs-on: macos-latest
     steps:
-      - name: Set up Go 1.19
+      - name: Set up Go 1.20
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: 1.20.0
```

Comment:
Is it worth using `1.20.x` instead, so that we automatically get patch versions? I'm not aware of there ever being anything in a patch version which we've had to model.